src/queryResult/queryResultWebViewController.ts

```diff
@@ -100,6 +100,61 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
                 message.selection,
             );
         });
+        this.registerRequestHandler("copySelection", async (message) => {
+            sendActionEvent(
+                TelemetryViews.QueryResult,
+                TelemetryActions.CopyResults,
+                {
+                    correlationId: this._correlationId,
+                    format: message.format,
```

Comment:
This message doesn't have format field, and we also need to remove the TODOs as they are for SaveAs features